src/core/qgsvectorfilewriter.cpp

```diff
@@ -982,6 +982,48 @@ QMap<QString, QgsVectorFileWriter::MetaData> QgsVectorFileWriter::initMetaData()
                          )
                        );
 
```

Comment:
Good point.
Maybe the information could be shared one day via GDAL_DATA and QGIS could read and send it to the translation machine at build time?